lerna.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
lerna and yarn workspaces ?

Reply:
@JAdshead https://github.com/americanexpress/holocron/blob/main/lerna.json

Reply:
that does not explain why it's needed. that could be legacy.

Reply:
@JAdshead it isn't legacy, `lerna` is still required to have a single command at the root level for releasing. https://github.com/americanexpress/holocron/blob/main/package.json#L26-L28